db/records/operations/insert.py

```diff
@@ -35,9 +35,9 @@ def insert_records_from_csv(table, engine, csv_filename, column_names, header, d
             formatted_columns = sql.SQL(",").join(
                 sql.Identifier(column_name) for column_name in column_names
             )
-
+            encoding = f"sql_{encoding}" if encoding == "ascii" else encoding
```

Comment:
We need to ensure the encoding option is supported by Postgres. Here's the list of supported options: https://www.postgresql.org/docs/9.3/multibyte.html

I tried with a CSV encoded with `utf_16_le`, which failed with the following error:

```
argument to option "encoding" must be a valid encoding name
LINE 1: ...) FROM STDIN CSV HEADER DELIMITER E','  QUOTE '"' ENCODING '..
```

Here's a test CSV you can use: [movies.csv](https://github.com/centerofci/mathesar/files/7281554/movies.csv)

FYI, the encoding is only detected correctly with `charset-normalizer` installed.